templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +26,12 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                  <a href="{%  url "taxi:manufacturer-update" pk=manufacturer.id %}">UPDATE</a>
+              </td>
+              <td>
+                  <a href="{%  url "taxi:manufacturer-delete" pk=manufacturer.id %}">DELETE</a>
```

Comment:
Code Style Issue: According to the checklist, it is recommended to add a `Cancel` button apart from the `Delete` button. The `Cancel` button should lead to the previous page the user was on.